macaroonbakery/tests/test_utils.py

```diff
@@ -21,7 +20,8 @@ def test_cookie_expires_naive(self):
             c.expires, int((timestamp - datetime(1970, 1, 1)).total_seconds()))
 
     def test_cookie_expires_with_timezone(self):
-        timestamp = datetime.now(pytz.UTC)
+        from datetime import tzinfo
+        timestamp = datetime.utcnow().replace(tzinfo=tzinfo())
```

Comment:
This works? tzinfo is documented as an abstract base class. You could use datetime.timezone if you have modern Python 3 instead. I guess it doesn't matter, as passing in a real tzinfo instance would only be testing that pyrfc3339's parse method works as documented